framework_tvb/tvb/tests/framework/core/services/hpc_scheduler_client_test.py

```diff
@@ -179,11 +180,12 @@ def test_prepare_inputs_with_eeg_monitor(self, operation_factory, simulator_fact
         dao.store_entity(prj_db_db)
 
         eeg_monitor = EEGViewModel(projection=proj.gid, sensors=sensors.gid)
+        eeg_monitor.region_mapping = region_mapping.gid
 
         sim_folder, sim_gid = simulator_factory(op=op, monitor=eeg_monitor)
         hpc_client = HPCSchedulerClient()
         input_files = hpc_client._prepare_input(op, sim_gid)
-        assert len(input_files) == 10
+        assert len(input_files) == 13
```

Comment:
I don't agree with this change. The test should be adjusted in such a manner that it still prepares 10 input files.
I agree with adding a RegionMapping since it is now required on the ProjMonitorViewModel, but it should be linked with the Connectivity and Surface we put on the simulator. 
By using RegionMappingIndex factory, without linking it to the connectivity and surface used on the simulator, we generate extra Connectivity and Surface H5 files in the input_files list.